components/raftstore-v2/src/operation/ready/snapshot.rs

```diff
@@ -373,10 +399,23 @@ impl<EK: KvEngine, ER: RaftEngine> Storage<EK, ER> {
         false
     }
 
+    pub fn report_snapshot(&self, peer_id: u64, status: raft::SnapshotStatus) {
```

Comment:
There's no second caller for this. And it should only be called when the snapshot is sent. So just inline the code into the caller is better.

Reply:
good catch, I will set it as private.